### PR TITLE
Add sectionsharingan aa1 to master

### LINKED ISSUE #24: There should be a complete graphical update to the SectionSharingan
**Is your feature request related to a problem? Please describe.**
It would be nice to have a full overhaul for the old SectionSharingan to implement the Vaporwave style and to fit our current standards.

**Describe the solution you'd like**
Update the look of the SectionSharingan, so that it presents the Vaporwave style.

**Additional context**
SectionSharingan

### PR DESCRIPTION
**Pull request by petschna**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the SectionSharingan is outdated and does not present the Vaporwave style. #24 

**What is the new behavior (if this is a feature change)?**
- Now there is a SectionSharingan which shows the Vaporwave style and fits the rest of the website with the overhauled design. Closes #24

### CODE REVIEW

**Review by petschna — COMMENTED** [review]
Should be ok to merge.

**Review by petute — APPROVED** [review]
Should be ok to merge.

File: src/components/organisms/SectionSharingan/index.jsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+              <MDBBox class="c">
+                <MDBBox class="iris iris-1">
+                  <MDBBox class="mitsudomoe mitsudomoe-1">
+                    <img src={Mitsudomoe1IMG} alt="mitsudomoe1" />
+                  </MDBBox>
+                </MDBBox>
+                <div class="sculptures">
+                  <img class="sculpture-overlay1" src={sculpture1}></img>
+                  <img class="sculpture-overlay2" src={sculpture2}></img>
+                </div>
```
Comment (kleberbaum) [inline src/components/organisms/SectionSharingan/index.jsx]:
```suggestion
                <div class="sculpture-overlay">
                  <img class="sculpture-img1" src={sculpture1}></img>
                  <img class="sculpture-img2" src={sculpture2}></img>
                </div>
```

File: src/components/organisms/SectionSharingan/index.jsx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+              <MDBBox class="c">
+                <MDBBox class="iris iris-2">
+                  <MDBBox class="mitsudomoe mitsudomoe-2">
+                    <img src={Mitsudomoe2IMG} alt="mitsudomoe2" />
+                  </MDBBox>
+                </MDBBox>
+                <div class="sculptures">
+                  <img class="sculpture-overlay1" src={sculpture1}></img>
+                  <img class="sculpture-overlay2" src={sculpture2}></img>
+                </div>
```
Comment (kleberbaum) [inline src/components/organisms/SectionSharingan/index.jsx]:
```suggestion
                <div class="sculptures-overlay">
                  <img class="sculpture-img1" src={sculpture1}></img>
                  <img class="sculpture-img2" src={sculpture2}></img>
                </div>
```

File: src/components/organisms/SectionSharingan/index.jsx [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+              <MDBBox class="c">
+                <MDBBox class="iris iris-3">
+                  <MDBBox class="mitsudomoe mitsudomoe-3">
+                    <img src={Mitsudomoe3IMG} alt="mitsudomoe3" />
+                  </MDBBox>
+                </MDBBox>
+                <div class="sculptures">
+                  <img class="sculpture-overlay1" src={sculpture1}></img>
+                  <img class="sculpture-overlay2" src={sculpture2}></img>
+                </div>
```
Comment (kleberbaum) [inline src/components/organisms/SectionSharingan/index.jsx]:
```suggestion
                <div class="sculptures-overlay">
                  <img class="sculpture-img1" src={sculpture1}></img>
                  <img class="sculpture-img2" src={sculpture2}></img>
                </div>
```

File: src/components/organisms/SectionSharingan/sharingan.scss [hunk 4]
```diff
@@ -0,0 +1,459 @@
...
+  position: relative;
+  display: inline;
+}
+
+.sculpture-overlay1 {
+  position: absolute;
+  width: 400px;
+  margin-left: 1100px;
+  margin-top: -150px;
+}
```
Comment (kleberbaum) [inline src/components/organisms/SectionSharingan/sharingan.scss]:
```suggestion
.sculpture-img1 {
  position: absolute;
  width: 400px;
  margin-left: 1100px;
  margin-top: -150px;
}
```

File: src/components/organisms/SectionSharingan/sharingan.scss [hunk 5]
```diff
@@ -0,0 +1,459 @@
...
+
+.sculpture-overlay2 {
+  position: absolute;
+  width: 400px;
+  margin-right: 400px;
+  margin-left: -360px;
+  margin-top: -450px;
+  transform: scaleX(-1);
+}
```
Comment (kleberbaum) [inline src/components/organisms/SectionSharingan/sharingan.scss]:
```suggestion
.sculpture-img2 {
  position: absolute;
  width: 400px;
  margin-right: 400px;
  margin-left: -360px;
  margin-top: -450px;
  transform: scaleX(-1);
}

```

File: src/components/organisms/SectionSharingan/sharingan.scss [hunk 6]
```diff
@@ -0,0 +1,459 @@
...
+  img:hover {
+    animation: rotate 700ms ease-in-out forwards;
+  }
+}
+
+
+.sculptures {
+  position: relative;
+  display: inline;
+}
```
Comment (kleberbaum) [inline src/components/organisms/SectionSharingan/sharingan.scss]:
```suggestion
.sculptures-overlay {
  position: relative;
  display: inline;
}
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by petschna — COMMENTED** [review]
Should be ok to merge.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**